lib/get_more_page_data.tsx

```diff
@@ -0,0 +1,31 @@
+import Router from 'next/router'
+
+export default async function getMorePageData(getDataParams) {
+  //getDataParams contains:
```

Comment:
https://google.github.io/styleguide/jsguide.html#jsdoc Let's use this convention for comments